client/src/components/StepWizard/WizardSubmit.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import { theme } from "constants/theme";
+import BaseButton from "components/Button/BaseButton";
+
+const { white, royalBlue } = theme.colors;
+const WizardSubmit = styled(BaseButton)`
+border-radius: 4rem;
+margin-right: 7rem;
+margin-left: 7rem;
+  cursor: pointer;
+  color: ${white};
+  padding: 0 0.1rem;
+  background-color: ${royalBlue};
+  &:hover,
+  &.am-button-active {
+    background-color: ${royalBlue};
+    color: ${white};
+  } 
+  html body &.am-button {
```

Comment:
I think this is the same as @idjevm used but I don't think this is much better than using !important
since this is just using 2 global selectors to increase the specificity...
anyhow, it's not that important here since I think the buttons aren't really extended in other places

Reply:
Alrighty then, let's merge this one and see if we get any feedback for new changes. but as @robinv85 this should be good for now then!